source/shared/cpp/ObjectModel/Enums.cpp

```diff
@@ -269,7 +271,8 @@ namespace AdaptiveCards
     DEFINE_ADAPTIVECARD_ENUM(ImageStyle, {
             {
                 {ImageStyle::Default, "default"},
-                {ImageStyle::Person, "person"}
+                {ImageStyle::Person, "person"},
+                {ImageStyle::RoundedCorners, "roundedCorners"}
```

Comment:
Why is roundedCorners a boolean everywhere but a style for images?

Reply:
It is coming from spec:
"We will also support rounded corners on Image elements, using the existing style property:  Default, Person and now RoundedCorners "